src/Components/Artwork/Save.tsx

```diff
@@ -240,6 +240,10 @@ export default createFragmentContainer(
       __id
       id
       is_saved
+      # There are instances where save icon is swapped when in Sale
+      sale {
+        is_closed
+      }
```

Comment:
This led to a series of `.graphql.ts` file updates below since Save is used in quite a few places.

Reply:
Ah yeah, you're totally right and i was mostly thinking in terms of the generated type; this isn't totally necessary here, will move directly into action buttons.

Reply:
Oh yea, true! I think that we probably just don't have much of that these days, but we should make sure we can handle that properly.